src/parsers/minecraft/coord/coord_base.ts

```diff
@@ -0,0 +1,129 @@
+import { CommandErrorBuilder } from "../../../brigadier_components/errors";
+import { StringReader } from "../../../brigadier_components/string_reader";
+import { ReturnHelper } from "../../../misc_functions";
+import { Parser, ReturnedInfo } from "../../../types";
+
+const MIXED = new CommandErrorBuilder(
+    "argument.pos.mixed",
```

Comment:
Can I just confirm that these are the same errors used in vanilla where possible?

Reply:
these are. I looked them up

Reply:
Cool, just wanted to double check. Thanks